gitpod-io/website
# Optimize some homepage images for modern browsers

## Description
* Optimized some images in place, to reduce their size - should make it faster for most (70% general population, probably 90%+ for devs) people
* Generated avif versions for some png images (IDEs) for modern browsers
* Used picture tag so browser knows when to use fallback image
* Reworked the link preloads to preload whats visible from the get go, and eager load (using native loading=eager) the rest of screenshots - should save some bandwidth on mobile
* Lazyload avatars in testimonials (and decode asynchronously) - far below the fold + some of them are hidden behind horizontal scroll - should make a positive difference on mobile

## How to test
* Go to homepage
* See if IDEs screenshots still work
* See if avatars in testimonials are present


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2010"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2010"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
